lib/resources/datav2/rest_v2.ts

```diff
@@ -30,7 +30,7 @@ export enum Adjustment {
   RAW = "raw",
   DIVIDEND = "dividend",
   SPLIT = "split",
-  BOTH = "both",
+  ALL = "all",
```

Comment:
This is a beaking change, I guess.

Reply:
It is actually a fix, not a change, it is `all` on the server side, not `both`